lib/bolt/outputter/human.rb

```diff
@@ -492,11 +492,11 @@ def print_module_list(module_list)
         end
       end
 
-      def print_plugin_list(plugin_list, modulepath)
+      def print_plugin_list(plugin_data)
```

Comment:
In general I'd kind of like to move towards named keyword args for things like this so that it's more clear what the data is and what parameters the function takes, so here it'd be (similar to above) `plugins:, modulepath:` and then called `print_plugin_list(**plugin_data)`